content/blog/hpe-sustainability-insight-center-api-at-a-glance.md

```diff
@@ -0,0 +1,301 @@
+date: 2024-06-24T12:44:18.079Z
+author: Didier Lalli
+authorimage: /img/didier-lalli.png
+disable: false
+tags:
+  - hpe-greenlake-api
+---
+<style>
+li {
+   font-size: 27px;
+   line-height: 33px;
+   max-width: none;
+}
+</style>
+
+HPE GreenLake provides a new service called HPE Sustainability Insight Center that can assist you in obtaining detailed information about carbon emissions, energy consumption and the cost of the infrastructure that is managed by HPE GreenLake. In this blog post, I will explain how to extract data from this service in a programmatic way using cURL and the HPE Sustainability Insight Center API.
+
+## What is HPE Sustainability Insight Center?
+
+HPE Sustainability Insight Center is a service that runs on HPE GreenLake. I can add it into my workspace from the HPE GreenLake catalogue under the Management & Governance category.
+
+![HPE GreenLake catalogue](/img/sic-blog-1.jpg "HPE GreenLake catalogue")
+
+Once deployed, the service provides a dashboard, which I can use to monitor carbon emissions, energy consumption, and energy cost. You can get more details about HPE Sustainability Insight Center from the [User Guide](https://support.hpe.com/hpesc/public/docDisplay?docId=a00120892en_us&page=GUID-64E2035A-138E-44E8-8A04-7968A272A97E.html).
+
+![HPE Sustainability Insight Center Console](/img/sic-blog-2.jpg "HPE Sustainability Insight Center Console")
+
+In addition to the dashboard, HPE Sustainability Insight Center provides an API, to enable programmatically accessing this same data.
+
+## What can I do with the HPE Sustainability Insight Center API?
+
+Important use cases covered by the HPE Sustainability Insight Center API include:
+
+* **Achieving sustainability goals** — Use data retrieved from the HPE Sustainability Insight Center API to measure your organization's power consumption and carbon footprint. With this data, your organization can make data-infused, informed decisions to reduce its climate impact and ensure its IT assets operate in a way that meets regulatory and business environmental sustainability goals.
+* **Monitoring IT costs** — Use data available from the HPE Sustainability Insight Center API to rationalize the power consumption of IT assets operations for cost efficiency. This will free up budget for innovative and growth-orientated investments.
+* **Integrating into reporting workflows** — Use data retrieved from the HPE Sustainability Insight Center to incorporate into existing analytics, reporting, dashboards, and forecasting workflows to give your organization a robust understanding of its IT operations.
+
+## Where is the HPE Sustainability Insight Center API documented?
+
+The [API specifications](https://developer.greenlake.hpe.com/docs/greenlake/services/sustainability/public/) are found on the [HPE GreenLake developer portal](https://developer.greenlake.hpe.com/) along with other HPE GreenLake APIs. For HPE Sustainability Insight Center, there are 3 API calls available:
```

Comment:
Second hotlink for HPE GreenLake developer portal

Maybe I would connect readers to the /services page https://developer.greenlake.hpe.com/docs/greenlake/services/ 

Also, I think the spelling should be "HPE GreenLake Developer Portal".

Reply:
Ok merci.